src/main/java/mate/jdbc/resources/init_db.sql

```diff
@@ -0,0 +1,17 @@
+CREATE DATABASE `manufacturer_db`;
```

Comment:
please move this file to `src/main/resources` folder

Reply:
removed